Makefile

```diff
@@ -96,3 +96,10 @@ clean-build:
 	@rm -Rf ./.build
 	@rm -Rf ./test/bdd/docker-compose.log
 
+.PHONY: clean-test-keys
```

Comment:
Should this be part of `make all`?

Reply:
it can, I just added to support refreshing keys, I can add it to `make all`

Reply:
added